Source/multi.cpp

```diff
@@ -463,49 +465,49 @@ bool InitMulti(GameData *gameData)
 
 } // namespace
 
-void multi_msg_add(byte *pbMsg, BYTE bLen)
+void multi_msg_add(const byte *data, size_t size)
 {
-	if (pbMsg != nullptr && bLen != 0) {
-		tmsg_add(pbMsg, bLen);
+	if (data != nullptr && size != 0) {
+		tmsg_add(data, size);
 	}
 }

```

Comment:
I think we can get rid of the checks, or make them asserts. They're never false.

Reply:
Good catch, I'll do this as a follow up PR where I'll shave it down a bit to make it more clear that it will never be null.

Reply:
Here we go: https://github.com/diasurgical/devilutionX/pull/3009